translations/en.json

```diff
@@ -383,6 +383,28 @@
       }
     },
     "recovery": {
+      "progress": {
+        "change_email": "Try a different email",
```

Comment:
email *address*?

Reply:
I just copied what's in the designs: https://www.figma.com/design/HUysJAhv6cK6p1Pc81Fxaa/Matrix-Authentication-Service-(MAS)?node-id=5354-3370&t=lkEDnlQD00OwKSmK-4